sxg_rs/src/static/entrypoint.html

```diff
@@ -0,0 +1,18 @@
+<!--
+Copyright 2021 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!DOCTYPE html>
+<meta charset="utf-8">
+<iframe src="./prefetch.html"></iframe>
```

Comment:
Leave an HTML comment that this is to ensure that hitting 'reload' in the browser shows success again, and not failure due to the fallback URL.

Reply:
Done